test/test_valgrind.sh

```diff
@@ -143,9 +143,17 @@ for test in $TESTS; do
 	./test/test_memspace_highest_capacity)
 		FILTER='--gtest_filter="-*highestCapacityVerify*"'
 		;;
+	./test/test_pool_residency)
+		echo "- SKIPPED"
```

Comment:
instead of this `echo && continue` can't you just write `export UMF_ZE_LOADER_LIB_NAME=umf_ze_loopback`...?
// hence remove the TODO comment